peppy/project.py

```diff
@@ -473,7 +474,7 @@ def _del_if_in(obj, attr):
             for s in track(
                 self.samples,
                 description="Removing sample attributes",
-                disable=not self.is_sample_table_large,
+                disable=not all([self.is_sample_table_large, self.progressbar]),
```

Comment:
instead of `all([self.is_sample_table_large, self.progressbar])` just  `self.is_sample_table_large and self.progressbar` seems more clear

Reply:
Fixed